code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -546,12 +546,13 @@
 				to_chat(user, "<span class='warning'>You need 50 metal to produce a construct shell!</span>")
 		else if(istype(target, /obj/item/stack/sheet/plasteel))
 			var/obj/item/stack/sheet/plasteel/candidate = target
-			var/quantity = min(candidate.amount, uses)
-			uses -= quantity
-			new /obj/item/stack/sheet/runed_metal(T,quantity)
-			candidate.use(quantity)
-			to_chat(user, "<span class='warning'>A dark cloud eminates from you hand and swirls around the plasteel, transforming it into runed metal!</span>")
-			SEND_SOUND(user, sound('sound/effects/magic.ogg',0,1,25))
+			var/quantity = candidate.amount
+			if(candidate.use(quantity))
+				uses --
+				new /obj/item/stack/sheet/runed_metal(T,quantity)
+				candidate.use(quantity)
```

Comment:
why are you calling this twice

Reply:
ah shit, made that change to an if(use) after testing - good catch